libs/carrier-portal/ui/src/lib/member-policy/member-policy.component.html

```diff
@@ -44,8 +44,25 @@ <h3>Enrollment Group</h3>
           </td>
           <td>{{ t('relationships.' + enrollee.relationship) | titlecase }}</td>
           <td>
-            {{ enrollee.coverage_start | date : 'M/d/yy' }}
-            - {{ (enrollee.coverage_end | date : 'M/d/yy') ?? '' }}
+            {{ enrollee.coverage_start | date : 'M/d/yy' }} -
+            <ng-container
+              *ngIf="enrollee.coverage_end; then endGiven; else noEndGiven"
+            >
+            </ng-container>
+
+            <ng-template #endGiven>
+              {{ enrollee.coverage_end | date : 'M/d/yy' }}
+            </ng-template>
+
+            <ng-template #noEndGiven>
+              <ng-template
+                [ngIf]="
+                  (enrollee.coverage_start | date : 'yyyy') !== currentYear
```

Comment:
This will show the incorrect result around December when looking at next years renewals.

For example, if it's currently November/December 2023, and I have the following policies:
* Policy A, runs 2023-01-01 - (no end date), currently active
* Policy B, runs 2024-01-01 - (no end date), renewed and will become active in january

The UI as implemented would show me the following dates:
* For Policy A: 01/01/2023 - 
* For Policy B: 01/01/2024 - 12/01/2024
Which is not correct.

Please extract this into a pipe and write a spec with multiple test cases.